app/assets/stylesheets/show.scss

```diff
@@ -16,3 +15,23 @@
     padding-right: 5px;
   }
 }
+
+.embed-container {
+  position: relative;
```

Comment:
Properties should be ordered height, height, margin, overflow, padding-bottom, padding-top, position, width